src/CreditCardInput.js

```diff
@@ -86,12 +87,15 @@ export default class CreditCardInput extends Component {
     invalidColor: "red",
     placeholderColor: "gray",
     allowScroll: false,
+    disableAutoFocus: false,
   };
 
-  componentDidMount = () => this._focus(this.props.focused);
+  componentDidMount = () => {
+    if (!this.props.disableAutoFocus) this._focus(this.props.focused);
+  }
 
   componentWillReceiveProps = newProps => {
-    if (this.props.focused !== newProps.focused) this._focus(newProps.focused);
```

Comment:
You're absolutely right--it would cause some odd behavior. I think your solution to add an initialValues prop makes a lot more sense to populate the input without focussing.